db/seeds/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Movies = require('./data/movies');
+
+exports.seed = (Knex) => {
+  return Knex('movies').truncate()
+  .then(() => {
+    return Knex('movies').insert(Movies);
+  });
+};
```

Comment:
nit: When using ES6 arrow functions, it's really handy to shorthand functions that just return something, so in this case

```js
.then(() => {
  return Knex('movies').insert(Movies);
});
```
turns into 

```js
.then(() => Knex('movies').insert(Movies));
```

Technically we could shorthand the entire `exports.seed` function, but that doesn't do much for readability. For promise chains, though, using the arrow function shorthands makes things more readable in my opinion 😄

Reply:
I agree! That part was copy-pasted from Miyagi so maybe we should change it there too.